src/sass/main.scss

```diff
@@ -16,13 +16,15 @@
 @import './categories/categories.scss';
 @import './categories/categories-home.scss';
 @import './categories/ch-category.scss';
+@import './categories/category-image.scss';
+@import './categories/ch-category-image-selector.scss';
 
 html, body {
-  scrollbar-width: none;
-  -ms-overflow-style: none;
+  // scrollbar-width: none;
```

Comment:
Please remove commented code or leave a note as to why it is commented out.

Reply:
Removed.